Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.cs

```diff
@@ -4317,6 +4332,10 @@ private static Type TypeOfRhs(AssignmentRhs rhs) {
       }
     }
 
+    protected virtual void EmitStaticExternMethodQualifier(string qual, ConcreteSyntaxTree wr) {
```

Comment:
I  chose the name not based on what the Rust code generator needed, but based on the path on which this method is called in the SinglePassCodeGenerator, along with the arguments it is provided. Such a name has less chances to conflict with any other future method names (e.g. a possible EmitExternTypeQualifier)
```
if (!s.Method.IsStatic) {
...
} else if (s.Method.IsExtern(Options, out var qual, out var compileName) && qual != null) {
  EmitStaticExternMethodQualifier(qual, wr);  // Method named chosen because of the path.
```
On the light of this naming convention, please confirm if you would still prefer to change the name.

Reply:
Ah fair, I missed that context.